antenna-documentation/src/site/markdown/generators/sw360-update-generator-step.md

```diff
@@ -23,6 +23,9 @@ Add the following step into the `<generators>` section of your workflow.xml
         <entry key="auth.server.url" value="http://localhost:8080/authorization"/>
         <entry key="username" value="admin@sw360.org"/>
         <entry key="password" value="12345"/>
+        <entry key="proxy.enable" value="true"/>
+        <entry key="proxy.host" value="${proxyHost}"/>
```

Comment:
@bs-ondem do we have already a global way to configure `proxyHost` and `proxyPort`? Maye it would be better to take the information from there and add only the `proxy.enable` option for the `sw360-updater`?